apps/fxc-front/src/app/components/2d/planner-element.ts

```diff
@@ -50,6 +68,11 @@ export class PlannerElement extends connect(store)(LitElement) {
     this.units = state.units;
     this.duration = ((this.distance / this.speed) * 60) / 1000;
     this.isFreeDrawing = state.planner.isFreeDrawing;
+    this.track = currentTrack(state);
+    this.league = state.planner.league;
+    if (!state.planner.enabled) {
```

Comment:
There is no need to destroy the scorer here.
You should do this in [`disconnectCallback`](https://lit.dev/docs/components/lifecycle/#disconnectedcallback)